usr/src/uts/common/vm/vm_page.c

```diff
@@ -5685,7 +5685,8 @@ page_migrate(
 	}
 }
 
-#define	MAX_CNT	60	/* max num of iterations */
+static uint_t reclaim_maxcnt = 60; /* max total iterations */
```

Comment:
Yeah, although we haven't had a need to tune them yet. Are you suggesting they shouldn't be, or that I didn't make them tunable correctly? (Sorry, having a bit of trouble parsing the intent behind your question.)